### PR TITLE
AV-1802: Accessible datepicker

### PR DESCRIPTION
**Pull request by teravill**
Implemented an accessible datepicker for advanced search, dataset, dataset resource and organization.
image

### CODE REVIEW

File: ckan/ckanext/ckanext-ytp_main/ckanext/ytp/schemas/presets.json [hunk 1]
```diff
@@ -245,6 +245,30 @@
...
+    {
+      "preset_name": "date_ab-datepicker",
+      "values": {
+        "form_snippet": "date_ab-datepicker.html",
+        "display_snippet": "date.html",
+        "validators": "scheming_required isodate from_date_is_before_until_date convert_to_json_if_date",
+        "form_attrs": {
+          "data-datepicker": "",
+          "class": "form-control"
+        }
```
Comment (Zharktas) [inline ckan/ckanext/ckanext-ytp_main/ckanext/ytp/schemas/presets.json:258]: These are probably not required as it has its own template anyway

**Review by Zharktas — CHANGES_REQUESTED** [review]
The css file of datepicker should be in .gitignore and copied during gulp assuming its unmodified.

File: ckan/ckanext/ckanext-ytp_main/ckanext/ytp/schemas/presets.json [hunk 2]
```diff
@@ -245,6 +245,30 @@
...
+    {
+      "preset_name": "date_sysadmin_ab-datepicker",
+      "values": {
+        "form_snippet": "date_sysadmin_ab-datepicker.html",
+        "display_snippet": "date.html",
+        "validators": "scheming_required isodate from_date_is_before_until_date convert_to_json_if_date",
+        "form_attrs": {
+          "data-datepicker": "",
+          "class": "form-control"
+        }
```
[the remark this answers is not in the document]
Comment (teravill) [reply ckan/ckanext/ckanext-ytp_main/ckanext/ytp/schemas/presets.json]: Resolved in [8910a57](https://github.com/vrk-kpa/opendata/pull/1757/commits/8910a5741af49cdb1b52f82f15fee20e0e92071e)

File: ckan/ckanext/ckanext-ytp_main/ckanext/ytp/schemas/presets.json (hunk 1 above)
Comment (teravill) [reply ckan/ckanext/ckanext-ytp_main/ckanext/ytp/schemas/presets.json]: Resolved in [8910a57](https://github.com/vrk-kpa/opendata/pull/1757/commits/8910a5741af49cdb1b52f82f15fee20e0e92071e)

**Comment by beazil** [comment]
No more duplicate icons, but it would be nice that other datepicker would close when another is opened, also position of datepicker is little odd:
image

**Review by Zharktas — APPROVED** [review]

**Pull request merged**